src/components/body/header/Header.css

```diff
@@ -1,8 +1,261 @@
-.nav-bar{
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    align-items: center;
+.side-bar {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 64px;
+  background: #FFFFFF;
+  border: 1px solid gray; /*for exam after delete border*/
+}
+
+.menu {
+  position: relative;
+  width: 64px;
+  height: 100%;
+}
+
+.menu-active {
+  display: inline-block;
+  width: 18px;
+  top: 25%;
+  border: 2px solid #0F1C2A;
+  position: relative;
+}
+
+.menu-active::before, .menu-active::after {
+  content: '';
+  position: absolute;
+  width: 18px;
+  border: 2px solid #0F1C2A;
+  left: -2px;
+}
+
+.menu-active::before {
+  top: -9px;
+}
+
+.menu-active::after {
+  bottom: -9px;
+}
+
+.menu-list {
+  padding: 0;
+  position: absolute;
+  text-align: left;
+}
+
+.menu-list li {
+  list-style: none;
+}
+
+.header {
+  display: grid;
+  gap: 16px;
+  grid-template-columns: repeat(12, 1fr);
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+
+.logo-name {
+  display: flex;
+  grid-column-start: 1;
+  grid-column-end: 4;
+  min-width: 230px;
+  margin-left: 24px;
+  text-decoration: none;
+  color: #0F1C2A;
+}
+
+.title-header {
+  display: inline-block;
+  margin: 0 0 0 5px;
 }
 
+.my-studio {
+  color: #455FCE;
+  padding-left: 16px;
+  margin-left: 16px;
+  font-family: 'Apercu Pro';
+  font-style: normal;
+  font-weight: 600;
+  border-left: 1px solid #E3E1EC;
+  font-size: 16px;
+  line-height: 150%;
+  letter-spacing: 0.15px;
+  text-decoration: none;
+}
+
+.search-button {
+  display: flex;
+  grid-column-start: 9;
+  grid-column-end: 13;
+  justify-content: flex-end;
+  margin-right: 24px;
+  min-width: 592px;
+}
+
+.magnifier {
+  display: none;
+}
+
+.search {
+  width: 100%;
+  height: 32px;
+  border: 1px solid #E3E1EC;
+  border-radius: 8px;
+  background: #FFFFFF;
+  margin-right: 16px;
+  padding-inline: 6px;
+}
+
+.search::placeholder {
+  background-image: url('../../icons/Magnifier.svg');
+  background-repeat: no-repeat;
+  background-position: left;
+  text-indent: 20px;
+  color: #E3E1EC;
+}
+
+.login-link {
+  display: inline-block;
+}
+
+.section-button {
+  display: flex;
+  text-decoration: none;
+}
+
+.section-button div {
+  display: flex;
+  align-items: center;
+}
+
+.avatar-login {
+  display: none;
+}
+
+.button-login {
+  width: 115px;
+  height: 32px;
+  font-family: 'Apercu Pro';
+  background: #455FCE;
+  color: #EAEAEA;
+  border: none;
+  border-radius: 28px;
+  font-weight: 600;
+  font-size: 16px;
+  box-sizing: content-box;
+}
+
+.user-item {
+  display: flex;
+  align-items: center;
+}
+
+.user-button {
+  width: 141px;
+  border: 1px solid #455FCE;
+  border-radius: 28px;
+  font-family: 'Apercu Pro';
```

Comment:
'Apercu Pro' is not a standard font family. If you use such families you need declare font face with source to the font